### ISSUE TITLE
Tint and srcCompat

### ISSUE THREAD

**Issue by dovahkiin98** [Clans/FloatingActionButton#437]
Tint doesn't apply to icon
and app:srcCompat isn't recognized

**Comment by Croutonix** [comment]
Yes, because `FloatingActionButton` extends `android.widget.ImageButton` and not the support library equivalent. The solution around this is to tint your own icons by code...

**Comment by kolagit** [comment]
As stated, this library is not supported any longer.
To make it compatible with AndroidX,
All you need to do is :

update ImageView to use "androidx.appcompat.widget.AppCompatImageView",
update ImageButton to use "androidx.appcompat.widget.AppCompatImageButton"
(You have to fix one of super class constructor as AppCompatImageButton constructor
changed. getOpacity() method.)
It fixed the AndroidX compatibility issue for me.
